src/components/TopBar/TopBar.vue

```diff
@@ -37,7 +37,9 @@
 					<NcPopover v-if="conversation.description"
 						:focus-trap="false"
 						:delay="500"
+						:boundary="boundaryElement"
 						:popper-triggers="['hover']"
+						:placements="['bottom']"
```

Comment:
typo? and it's default one, so we can omit it: https://floating-vue.starpad.dev/api/#placement

Reply:
yeah, I didn't check if it is by default